src/CMakeLists.txt

```diff
@@ -60,13 +60,19 @@ if(nfd_PLATFORM STREQUAL PLATFORM_LINUX)
   endif()
 endif()
 
+# MacOS is deprecating the allowedFileTypes property in favour of allowedContentTypes, so we have
+# to introduce this breaking change. Set NFD_USE_ALLOWEDCONTENTTYPES to ON, to have it set the  
+# allowedContentTypes property of the SavePanel or OpenPanel if the executable is being built for
+# macOS 11.0 or higher.
```

Comment:
Can we move this comment to where we declare the option?  That's more likely to be where someone would expect documentation for it.